share/eds/blockstore.go

```diff
@@ -79,7 +83,16 @@ func (bs *blockstore) Get(ctx context.Context, cid cid.Cid) (blocks.Block, error
 	return nil, err
 }
 
+const maxDataSize = max(share_ipld.InnerNodeSize, share_ipld.LeafNodeSize)
```

Comment:
Leafnodesize is always bigger and its an axiom. Not sure why we would do the max op

Reply:
Quite obviously: to show that we select max of 2 possible values. Anyway, have changed to `return share_ipld.LeafNodeSize, nil`